libguard/guard_interface.cpp

```diff
@@ -153,7 +153,6 @@ GuardRecord create(const EntityPath& entityPath, uint32_t eId, uint8_t eType)
     //! check if guard record already exists
     int pos = 0;
     int lastPos = 0;
-    uint32_t maxId = 0;
```

Comment:
Why you are making changes in create() API? no information about this in the commit message?

Reply:
Will add in the commit msg. basically this required as we are not deleting the guard record we are just marking the recordID as FFFFFFF so the maxid will get set to that and we will not get the actual count present.